src/Containers/Note/Note.js

```diff
@@ -25,11 +25,32 @@ export class Note extends Component {
     } catch (e) { console.log(e) }
```

Comment:
It's so weird, I just pulled master and it's showing something completely different here.